src/client/containers/SignUp/index.js

```diff
@@ -44,6 +45,7 @@ export default function SignUpContainer() {
           linkedin,
           github,
           website,
```

Comment:
website is not in users signup form. can you cjeck this please

Reply:
it is in database diagram. i guess we have to ask the project lead if we should delete the website from database or if we should add website to user form. good find :)

Reply:
SInce it was not added is in this PR, we will just leave it as it for now. Cheers,